qan/analyzer/mongo/aggregator/aggregator_test.go

```diff
@@ -0,0 +1,114 @@
+package aggregator
+
+import (
+	"testing"
+	"time"
+
+	"github.com/percona/go-mysql/event"
+	"github.com/percona/percona-toolkit/src/go/mongolib/proto"
+	pc "github.com/percona/pmm/proto/config"
+	"github.com/percona/pmm/proto/qan"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAggregator_Add(t *testing.T) {
```

Comment:
As agreed on a standup call I will address this later - we need to merge this quickly so QA can test it.